doc/templates.rst

```diff
@@ -918,16 +918,14 @@ Extensions
 Twig can be extended. If you want to create your own extensions, read the
 :ref:`Creating an Extension <creating_extensions>` chapter.
 
-.. _`Twig bundle`:                https://github.com/Anomareh/PHP-Twig.tmbundle
-.. _`Jinja syntax plugin`:        http://jinja.pocoo.org/docs/integration/#vim
+.. _`Twig bundle`:                https://github.com/uhnomoli/PHP-Twig.tmbundle
```

Comment:
I updated the Twig bundle URL because the user changed its username 

https://github.com/Anomareh/PHP-Twig.tmbundle

--> redirect to

https://github.com/uhnomoli/PHP-Twig.tmbundle